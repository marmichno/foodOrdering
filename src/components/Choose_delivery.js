import {Link} from 'react-router-dom';

export const Choose_delivery = () => {

    return(
        <div className="chooseDeliveryContainer">
                <div className="chooseDeliveryTextContainer">
                    <h1>Choose delivery option</h1>
                    <Link to={{
                        pathname:'/order/choose_delivery/delivery_details'
                    }}>
                        <button>I will pick it up</button>
                    </Link>
                    <Link to={{
                        pathname:'/order/choose_delivery/delivery_details'
                    }}>
                        <button>I want to order with delivery</button>
                    </Link>
                </div>
        </div>
    )
}