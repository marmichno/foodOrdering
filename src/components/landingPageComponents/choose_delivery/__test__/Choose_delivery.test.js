import {Choose_delivery} from '../Choose_delivery';
import {BrowserRouter} from 'react-router-dom';
import TestRenderer from "react-test-renderer";

test("matches snapshot", () => {
    const tree = TestRenderer.create(<BrowserRouter><Choose_delivery/></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
});