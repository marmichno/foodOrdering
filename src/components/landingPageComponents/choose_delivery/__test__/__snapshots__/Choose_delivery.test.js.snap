// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="chooseDeliveryContainer"
>
  <div
    className="chooseDeliveryTextContainer"
    style={
      Object {
        "transform": "translateX(2000px) translateZ(0)",
      }
    }
  >
    <h1>
      Choose delivery option
    </h1>
    <button
      onClick={[Function]}
    >
      I will pick it up
    </button>
    <button
      onClick={[Function]}
    >
      I want to order with delivery
    </button>
  </div>
</div>
`;
